src/main/java/me/jellysquid/mods/lithium/mixin/ai/task/MixinFollowMobTask.java

```diff
@@ -0,0 +1,42 @@
+package me.jellysquid.mods.lithium.mixin.ai.task;
+
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.ai.brain.Brain;
+import net.minecraft.entity.ai.brain.MemoryModuleType;
+import net.minecraft.entity.ai.brain.task.FollowMobTask;
+import net.minecraft.server.world.ServerWorld;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.Shadow;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Predicate;
+
+@Mixin(FollowMobTask.class)
+public class MixinFollowMobTask {
+    @Shadow
+    @Final
+    private Predicate<LivingEntity> predicate;
+
+    /**
+     * @reason Replace stream code with traditional iteration
```

Comment:
perhaps a forindex loop will be even faster 😉

Reply:
An iterator is the fastest way to create a loop for a List.